ContractsApi/V1/Factories/EntityFactory.cs

```diff
@@ -84,7 +85,8 @@ public static ContractDb ToDatabase(this Contract contract)
                 Rates = contract.Rates,
                 DefaultTenureType = contract.DefaultTenureType,
                 SuspensionDate = contract.SuspensionDate,
-                ReasonForSuspensionDate = contract.ReasonForSuspensionDate
+                ReasonForSuspensionDate = contract.ReasonForSuspensionDate,
```

Comment:
Looks like the factory tests haven't been updated for a while, so they are missing a lot of properties. Would be good to get them up to date and include this prop too